apps/web/src/components/templates/TriggerSnippetTabs.tsx

```diff
@@ -9,7 +14,13 @@ import * as get from 'lodash.get';
 const NODE_JS = 'Node.js';
 const CURL = 'Curl';
 
-export function TriggerSnippetTabs({ trigger }: { trigger: INotificationTrigger }) {
+export function TriggerSnippetTabs({
+  trigger,
+  template,
```

Comment:
Instead of adding template to props and the multiple changes that follows that change, might be better to just use getValues of steps. It will not cause rerenders :) 

Something like this:

<img width="1011" alt="Screen Shot 2023-02-19 at 17 46 18" src="https://user-images.githubusercontent.com/7778680/219958688-55dd96f9-0e85-476d-b7ef-c736ee4e4e65.png">